sale_company_currency/__manifest__.py

```diff
@@ -0,0 +1,18 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Denis Leemann - Camptocamp SA
```

Comment:
Copyright is attributed to the company: `Copyright 2017 Camptocamp SA`. The author can be found in the readme and in git.